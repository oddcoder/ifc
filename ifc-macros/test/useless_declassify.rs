use ifc_macros::ifc_block;

fn main() {
    ifc_block!{
        #[IFC(Low, Declassify)]
        let _a = 5;
        #[IFC(Low, Declassify)]
        let _x = 4;
    }
    compile_error!("");
}