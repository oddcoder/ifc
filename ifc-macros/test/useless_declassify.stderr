warning: Unused attribute
 --> test/useless_declassify.rs:5:20
  |
5 |         #[IFC(Low, Declassify)]
  |                    ^^^^^^^^^^
  |
  = help: Try removing it

warning: Unused attribute
 --> test/useless_declassify.rs:7:20
  |
7 |         #[IFC(Low, Declassify)]
  |                    ^^^^^^^^^^
  |
  = help: Try removing it

error:
  --> test/useless_declassify.rs:10:5
   |
10 |     compile_error!("");
   |     ^^^^^^^^^^^^^^^^^^
